MQTicketService/TicketGenerator/com/ibm/mq/badge/EnvSetter.java

```diff
@@ -124,8 +124,10 @@ public JMSContext connect() {
             }
             catch (JMSRuntimeException e) {
```

Comment:
Clean up the error catch. Currently the code is
```java
e.printStackTrace();
logger.log(Level.SEVERE, "Credentials are incorrect, could not create context", e);
```
when the error is instanceof JMSSecurityRuntimeException as well as when it is not. Decide which it should be and remove the 2 duplicated lines.

Clean up the indentation in the method setJMSProperties